docs/metalava.md

```diff
@@ -0,0 +1,37 @@
+## [Metalava](https://android.googlesource.com/platform/tools/metalava/) setup
+
+Use `$> make 1.0-core-check-api` `$> 1.0-core-update-api` (Core) `$> make 1.0-ui-check-api` `$> 1.0-ui-update-api` (UI) from the [`Makefile`](https://github.com/mapbox/mapbox-navigation-android/blob/master/Makefile) to interact.
+
+#### When releasing
+
+Remember to copy `api/current.txt` files to `api/X.Y.Z.txt` for every module
```

Comment:
Is this automatic? Or is this adding a release step?

Reply:
For now, this is adding a new release step - until we work on automating the release process 🚀